io/io/test/TFileTests.cxx

```diff
@@ -130,6 +130,7 @@ void TestReadWithoutGlobalRegistrationIfPossible(const char *fname)
 // https://github.com/root-project/root/issues/10742
 #ifndef R__WIN32
 // We prefer not to read remotely files from Windows, if possible
+#ifdef R__DAVIX_SUPPORT
```

Comment:
This is indeed the solution, however I would propose we go for the existing macro `R__HAS_DAVIX`